Actions/ReadSettings/ReadSettings.ps1

```diff
@@ -66,6 +66,10 @@ $settings.Keys | ForEach-Object {
     }
 }
 
+if('CICDPushBranches' -in $getSettings -and 'CICDPushBranches' -notin $settings.Keys) {
```

Comment:
This is done because `CICDPushBranches` is not in the settings by default. Add a comment to explain it.